ac/ac-plot/src/index.js

```diff
@@ -33,32 +33,36 @@ const config = {
 
 const dataStructure = {};
 
-const mergeFunction = ({ data: incoming }, dataFn, data) => {
-  if (Array.isArray(incoming)) {
-    incoming.forEach(item => mergeFunction({ data: item }, dataFn, data));
-    return;
-  }
-  if (isEmpty(incoming) || !isObject(incoming)) {
-    return;
-  }
-  if (incoming['1']) {
-    mergeFunction({ data: incoming['1'] }, dataFn, data);
-  }
-  if (incoming.y) {
-    if (!data[incoming.trace]) {
-      dataFn.objInsert({ y: [] }, incoming.trace);
```

Comment:
I believe this was a bug. 
Does the `data` object changes synchronously when a dataFn function is called? I think it does not. This would then create bugs if we insert several points as we reinitialize the dataset for each point.